frame/3/gemm/bli_gemm_threading.c

```diff
@@ -112,13 +112,27 @@ void bli_gemm_thrinfo_free_paths( gemm_thrinfo_t** threads, dim_t num )
 gemm_thrinfo_t** bli_create_gemm_thrinfo_paths( )
 {
 
-#ifdef BLIS_ENABLE_MULTITHREADING 
+#ifdef BLIS_ENABLE_MULTITHREADING
     dim_t jc_way = bli_read_nway_from_env( "BLIS_JC_NT" );
-//    dim_t kc_way = bli_read_nway_from_env( "BLIS_KC_NT" );
+    // dim_t kc_way = bli_read_nway_from_env( "BLIS_KC_NT" );
     dim_t kc_way = 1;
     dim_t ic_way = bli_read_nway_from_env( "BLIS_IC_NT" );
     dim_t jr_way = bli_read_nway_from_env( "BLIS_JR_NT" );
     dim_t ir_way = bli_read_nway_from_env( "BLIS_IR_NT" );
+    char* str = getenv( "OMP_NUM_THREADS" );
+    if( str != NULL )
+    {
+       int max_threads = strtol( str, NULL, 10 );
+       max_threads = max( max_threads, 1 );
```

Comment:
@poulson `max` isn't an intrinsic in C.  You'll need to add a macro or just use the ternary here.